release/goreleaser.opm.Dockerfile

```diff
@@ -1,11 +1,16 @@
 # NOTE: This Dockerfile is used in conjuction with GoReleaser to
 #   build opm images. See the configurations in .goreleaser.yaml
 #   and .github/workflows/release.yaml.
+FROM alpine as fsutil
+RUN mkdir -p /opm
```

Comment:
Is it necessary to do this with a separate image? I thought the `gcr.io/distroless/static:debug` had busybox utils that could help us here.